File: sponge-jruby/examples/script/rb/triggers_hello_world.rb
# Sponge Knowledge base
# Hello world

class HelloWorldTrigger < Trigger
    def onConfigure
        self.withEvent("helloEvent")
    end

    def onRun(event)
        puts event.get("say")
    end
end

def onStartup
	$sponge.event("helloEvent").set("say", "Hello World!").send()
end

File: sponge-jython/examples/script/py/triggers_hello_world.py
"""
Sponge Knowledge base
Hello world
"""

class HelloWorldTrigger(Trigger):
    def onConfigure(self):
        self.withEvent("helloEvent")
    def onRun(self, event):
        print event.get("say")

def onStartup():
    sponge.event("helloEvent").set("say", "Hello World!").send()
